sky/engine/core/window/window.h

```diff
@@ -33,6 +33,8 @@ class Window {
 
   void DidCreateIsolate();
   void UpdateWindowMetrics(const SkyDisplayMetrics& metrics);
+  void UpdateLocale(const std::string& language_code,
+		    const std::string& country_code);
```

Comment:
The step where you `clang-format` the diff before the PR. I keep forgetting to do that.